tests/testsuite/cargo_add/git_multiple_packages_features/mod.rs

```diff
@@ -0,0 +1,43 @@
+use cargo_test_support::compare::assert_ui;
+use cargo_test_support::prelude::*;
+use cargo_test_support::Project;
+
+use cargo_test_support::curr_dir;
+
+#[cargo_test]
+fn case() {
+    cargo_test_support::registry::init();
+
+    let mut members_manifest = cargo_test_support::basic_manifest("packmain", "0.1.1+mainpack");
+    members_manifest.push_str("members = [pack1, pack2]");
```

Comment:
I feel like this is brittle, assuming that a `[workspace]` is placed last.

Personally, I'd just go ahead and write out the desired manifest.